### PR TITLE
Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by pulgosodog**
Changes made acodring to the Portfolio: add a contact form and finish the mobile version activity:

Add a form with name, email, and text area fields.

Update style.css to style tags in contact form section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,6 +233,22 @@ <h3 class="margin">About Myself</h3>
             </li>
         </ul>
     </section>
+    <section class="contact-form">
+        <div class="blue-border flex-column">
+        <h2 class="margin">Contact me</h2>
+        <p class="margin">If you have an application you are interested in developing, a feature that you need built or
+            a project that needs coding.
+            I’d love to help with it!</p>
+            <form class="margin" id="survey-form" action="" method="post">
```
Comment (Ibizugbe) [inline index.html:242]: Just as required for this milestone, kindly integrate your form with `formspre` service so as to retrieve user data from the form👍.
FORMSPRE GITHUB

image

File: index.html [hunk 2]
```diff
@@ -232,6 +233,22 @@ <h3 class="margin">About Myself</h3>
...
+                <input type="text" id="fname" name="name" maxlength="30" required placeholder="Lorem Ipsum"><br>
+                <input type="email" id="email" required placeholder="loremipsum@gmail.com"><br>
+                <textarea maxlength="500" rows="4" cols="50" required
+                    placeholder="Write your message here..."></textarea>
+                <button type="submit">Get in touch</button>
+            </form>
+        </div>
+        <hr>
+    </section>
```
Comment (Ibizugbe) [inline index.html:251]: Kindly do well to add the background image to your form😉 as it is required that we stick to the given design in this milestone👍. Kindly take note of the highlighted parts of the design below😉😉.


image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @pulgosodog👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linters are passing👍.
- [x] Github flow have been used👍.
- [x] Input fields have the correct type attributes👍.
- [x] Validations have been written for the form👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (pulgosodog) [reply index.html]: Addressed!

File: index.html (hunk 2 above)
Comment (pulgosodog) [reply index.html]: Fixed! Changes applied.

**Review by Ibizugbe — APPROVED** [review]
Hi @pulgosodog 👋👋👋, 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅

- [x] Linters are passing👍.
- [x] Form have been integrated with formspre service👍.
- [x] Form section have been adjusted to match design👍.
- [x] Commit messages are descriptive and meaningful👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**